Source/Fuse.Nodes/AppConfig.uno

```diff
@@ -0,0 +1,23 @@
+namespace Fuse
+{
+    /** Allows configuration of App.Background through data bindings .
+
+        As the `App` class is not a `PropertyObject` and doesn't have a containing data context,
+        properties like `Background` cannot be databound regularly. This behavior class
+        allows you to do that from a specific data context.
+
+        ## Example
+
+            <App>
+                ...
+                <AppConfig Background="{foo}" />
+    */
+    public class AppConfig: Behavior
+    {
+        public float4 Background 
+        {
+            get { return AppBase.Current.Background; }
+            set { AppBase.Current.Background = value; }
```

Comment:
@yupferris: Well, this is kind of a hole we've dug ourselves into. The color really belongs to an implicit `GraphicsView` directly inside the App-tag. However, that is not exposed, so the user can't easily change that. That `App` has a `Background`-property is meant to be a way of exposing this to the API-user.